meerk40t/extra/inkscape.py

```diff
@@ -218,3 +227,253 @@ def inkscape_base(**kwargs):
             root_context = kernel.root
             root_context.setting(str, "inkscape_path", "inkscape.exe")
             return "inkscape", (root_context.inkscape_path, None)
+
+
+def register_inkscape_conversion(kernel):
+
+    import wx
```

Comment:
Why is this a wx required extension? 

Get translation and the dialog don't seem like they would be at all required for this to work.